Products/PluggableAuthService/events.py

```diff
@@ -40,6 +45,22 @@ def __init__(self, principal):
         self.object = principal
 
 
+@implementer(IPrincipalAddedToGroupEvent)
+class PrincipalAddedToGroup(PASEvent):
+
+    def __init__(self, principal, group):
```

Comment:
I would change this to principal_id and group_id to make it clear it is a string and not an object. The same change should be also done in `PrincipalRemovedFromGroup` and in the ifaces